packages/config/test/project/utils.spec.ts

```diff
@@ -0,0 +1,1182 @@
+import '@packages/server/test/spec_helper'
+import { getCtx } from '@packages/data-context/src/globalContext'
```

Comment:
We need `globalContext` here. Otherwise, we need to import the entire `pkg/data-context`.